tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -390,6 +411,7 @@ object QualOutputWriter {
 
   def getDetailedHeaderStringsAndSizes(appInfos: Seq[QualificationSummaryInfo],
       reportReadSchema: Boolean): LinkedHashMap[String, Int] = {
+    val clustertags = appInfos.exists(_.clusterTags.nonEmpty)
```

Comment:
nit camel case clusterTags or just remove and put  appInfos.exists(_.clusterTags.nonEmpty) in if statement below